### PR TITLE
Find data - images #4a7q30

### PR DESCRIPTION
**Pull request by cameronbaney**
# Description

The purpose of this PR is to add functionality for the images tab on the Find Data page.

Notable missing features:
- Empty state for no filters
- Ability to download a file

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- Go to the Find Data page
- Click on the Images tab
- You should see search results for the images, and all search functionality should work
- Click on a file's menu button, and select "Open Dataset"
- You should be taken to that dataset's landing page

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: pages/data/index.vue [hunk 1]
```diff
@@ -287,15 +285,67 @@ export default {
...
+
+    /**
+     * Get Search results
+     * This is using fetch from the Blackfynn API
+     */
+    fetchFromBlackfynn: function() {
+      this.isLoadingSearch = true
+
+      const searchType = pathOr('', ['query', 'type'], this.$route)
+      let url = `${process.env.discover_api_host}/search/${searchType}s?offset=${this.searchData.skip}&limit=${this.searchData.limit}&organization=SPARC%20Consortium`
```
Comment (cameronbaney) [inline pages/data/index.vue]: @mahamzayed These are the changes to this conditional URL I mentioned in #21

File: pages/data/index.vue [hunk 2]
```diff
@@ -287,15 +285,67 @@ export default {
...
+
+      if (searchType === 'file') {
+        url += '&fileType=tiff'
+      }
+
+      const query = pathOr('', ['query', 'q'], this.$route)
+      if (query) {
+        url += `&query=${query}`
+      }
+
```
Comment (mahamzayed) [inline pages/data/index.vue]: I see, this is why you can't make it a computed property, because of the query bit.

Comment (mahamzayed) [reply pages/data/index.vue]: Though I still feel like this logic could still be implemented in a computed prop, you'd just return a different url based on whether the `searchType` or `query` condition applies.

**Comment by cameronbaney** [comment]
True, good points. I wasn't thinking it'd be two different computed
properties. I was trying to take a more functional approach to this but I
can make that change.

File: pages/data/index.vue (hunk 2 above)
Comment (cameronbaney) [reply pages/data/index.vue]: @mahamzayed I moved this to a computed property, and all tests are passing.

**Pull request merged**